scheduler/src/cook/scheduler/scheduler.clj

```diff
@@ -245,7 +245,10 @@
                job-resources (util/job-ent->resources job-ent)
                pool-name (util/job->pool-name job-ent)
                ^TaskScheduler fenzo (get pool->fenzo pool-name)]
-           (when (#{:instance.status/success :instance.status/failed} instance-status)
+           ; Q: How to disable this for kubernetes only. In the case of kubernetes, this tells k8s that the resources can be used almost immediately.
```

Comment:
I had some odd kubernetes overallocation failures that seemed linked to this code, so I want it disabled for kubernetes.

But, this seems wrong to me for mesos too. It seems we should delete it entirely. However, I don't fully understand why it exists, so I'm opening it up to discussion. If we can figure out what to do next, I'll respin the patch appropriately.

Any idea why this unassign code exists?

Reply:
Paul mentiosn that we probably want this code for kubernetes too. For fenzo constraints to work, fenzo needs to track all running jobs to handlethings like, e.g., constraints, or running a group of jobs on different machines. We need to validate that we load everything into fenzo on startup in this code. We may or may not be doing so.